src/components/Map/geo-albers-usa-territories.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'geo-albers-usa-territories';
```

Comment:
this module doesn't have typings, so declaring this got me past some errors, but I'd be happy to find a better approach.